### DISCUSSION TITLE
NASA ESDIS System Engineering Technical Interchange Meeting (SE TIM): Proposal submission?

### DISCUSSION THREAD

**Discussion by asteiker** [earthaccess-dev/earthaccess#604] [General]
The ESDIS System Engineering Technical Interchange Meeting (SE TIM) is accepting proposals for their upcoming meeting Aug 21-22. A wiki page is up with more information, though note this may not be fully public: https://wiki.earthdata.nasa.gov/pages/viewpage.action?spaceKey=EM&title=ESDIS+SE+TIM+-+August+2024
Proposal submissions are due June 26th and I think earthaccess would be an excellent topic, especially aligning with this year’s theme:

_Faster, cheaper, leaner: quickly delivering value and responding to the needs of our users_

Here is the direct form submission: https://docs.google.com/forms/d/e/1FAIpQLSftfE83-gzlr8o9N2jSizIFIqx9kg-doAe1ZTiV29g-7mqz1A/viewform. Some folks at NSIDC DAAC are starting to coordinate and talk about submissions and I suggested earthaccess. Are others interested in contributing?

**Comment by mfisher87** [comment]
Heck yeah!

**Comment by asteiker** [comment]
There are a few different session types that we can propose:

* Classic 15-20 minute talk or panel presentation, etc.
* Working Session -small group breakout sessions used to accelerate progress toward reaching targeted goals in a small amount of time
* Lightning Talk (10 minute presentation grouped together with like topics)

I'm wondering if we want to propose a working session or classic session. Given this year's theme, I'd love for us to focus on success stories related to how we respond to and work _with_ our user community to provide value.

**Reply by mfisher87** [reply]
>how we respond to and work with our user community to provide value.

A working session sounds like an amazing way to demonstrate this!

**Reply by asteiker** [reply]
@mfisher87 Let me know what you think of my abstract draft! https://docs.google.com/document/d/1-KMGjyNo1VxjG3wMqAJJk6A-RyN7IIjdl45YFDpwXms/edit?usp=sharing.

**Reply by mfisher87** [reply]
Love it! It's concise and clear. No notes :)

**Comment by asteiker** [comment]
I started a draft agenda for our 30-minute discussion session: https://docs.google.com/document/d/1-KMGjyNo1VxjG3wMqAJJk6A-RyN7IIjdl45YFDpwXms/edit#heading=h.bw3pqzb3xuf7

@jhkennedy You mentioned you will be at the SE TIM in person. Would you like to co-facilitate this with me in-person? @danielfromearth I believe you will also be at the meeting, though I know you are also leading other sessions. Who else would like to contribute either in-person or online?

Please feel free to add ideas in the doc (please request edit access) and we can continue to build this out. I'd love to discuss in a breakout at next week's hackday, though I can only make the second hour.
